src/shared/js/table-stacked-mixin.js

```diff
@@ -0,0 +1,19 @@
+const tableStackedMixin = {
+  /**
+   * Handle attribute propagation for a stacked table.
+   *  codElement.handleTableStacked(
+   *    this, // the custom element.
+   *    node, // some other element lower in the DOM tree.
+   *  )
+   */
+  handleTableStacked(currentHtmlElement, targetHtmlElement) {
```

Comment:
Note: We can likely remove this mixin entirely since state propagation is also handled in `observedAttributeMixin.handleObservedClassAttribute` and [observed attribute handlers are called after the attribute is initialized, when the element's declaration is parsed for the first time](https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_custom_elements#responding_to_attribute_changes) if the element's HTML declaration includes an observed attribute.

I'll need to test this though since it's also possible the element is not defined until later (after the element declaration is parsed) during element upgrade and it's not clear what happens in that case.